src/metric-engine/Cargo.toml

```diff
@@ -9,7 +9,6 @@ workspace = true
 
 [dependencies]
 api.workspace = true
-aquamarine.workspace = true
```

Comment:
We use this dep to generate docs.
https://github.com/GreptimeTeam/greptimedb/pull/5181

Reply:
will bring it back maybe to dev?

Reply:
`dev-dependencies` may not work.
https://github.com/mersinvald/aquamarine/blob/master/demo/Cargo.toml